scripts/bootstrap.js

```diff
@@ -101,12 +98,24 @@ function run() {
       defaultValue: false,
       option: '--build',
       command: () => {
-        log.info(prefix, 'prepare');
-        spawn(
-          `nx run-many --target="prepare" --all --parallel ${
-            process.env.CI ? `--max-parallel=${maxConcurrentTasks}` : ''
-          } -- --optimized`
-        );
+        log.info(prefix, 'build');
+        const command = 'prepare';
+        const parallelism = process.env.CI ? `--max-parallel=${maxConcurrentTasks}` : '';
+
+        spawn(`nx run-many --target="${command}" --all --parallel ${parallelism}`);
+      },
+      order: 2,
+    }),
+    publish: createTask({
+      name: `Prepublish packages ${chalk.gray('(publish)')}`,
+      defaultValue: false,
+      option: '--publish',
+      command: () => {
+        log.info(prefix, 'publish');
+        const command = 'prepublish';
+        const parallelism = process.env.CI ? `--max-parallel=${maxConcurrentTasks}` : '';
+
+        spawn(`nx run-many --target="${command}" --all --parallel ${parallelism}`);
       },
       order: 2,
     }),
```

Comment:
This is never really supposed to be run, unless you're testing the production `dist`

Reply:
@shilman I imagine you might run this prior to publishing, WDYT, will you adjust your publish script?